src/pages/filter/page.tsx

```diff
@@ -53,6 +53,12 @@ const FilterPage = observer((): ReactElement => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [dsName, history])
 
+  const { statAmount } = datasetStore
+
+  const variantCounts = statAmount?.[0] ?? null
```

Comment:
You use this code many times.
May be we can create getters in datasetStore to get values something like this: `const { variantsCounts, dnaVariantsCounts, transcriptsCounts } = datasetStore`